sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -231,7 +231,7 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
   createQueryTest("no from clause",
     "SELECT 1, +1, -1")
 
-  if (!conf.ansiEnabled) {
+  if (!spark.sessionState.conf.ansiEnabled) {
```

Comment:
Kindly revert all modifications made to this test suite, as the test code is inconsequential to the performance evaluation.